README.md

```diff
@@ -424,28 +424,164 @@ The following behaviours apply to both `Multibranch Pipeline Jobs` and `Folder O
 
 * `Filter by name (with wildcards)` - To filter the type of items you want to discover in your project based on the wildcards specified. For example, to discover only `master` branch, `develop` branch and all Merge Requests add `development master MR-*`.
 
+* `Skip pipeline status notifications` - To disable notifiying GitLab server about the pipeline status.
+
+* `Override hook management modes` - Override default hook management mode of web hook and system hook.
+
+* `Checkout over SSH` - [Not Recommended] Use this mode to checkout over SSH. Use `Checkout Credentials` instead.
+
+## Job DSL seed job configuration
+
+To create a Job DSL seed job see this [tutorial](https://github.com/jenkinsci/job-dsl-plugin/wiki/Tutorial---Using-the-Jenkins-Job-DSL).
+
+Here is a sample seed job script for folder organisation job:
+
+```groovy
+organizationFolder('GitLab Organization Folder') {
+    description("GitLab org folder created with Job DSL")
+    displayName('My Project')
+    // "Projects"
+    organizations {
+        gitLabSCMNavigator {
+            projectOwner("baymac")
+            credentialsId("i<3GitLab")
+            serverName("gitlab-3214")
+            // "Traits" ("Behaviours" in the GUI) that are "declarative-compatible"
+            traits {
+                subGroupProjectDiscoveryTrait() // discover projects inside subgroups
+                gitLabBranchDiscovery {
+                    strategyId(3) // discover all branches
+                }
+                originMergeRequestDiscoveryTrait {
+                    strategyId(1) // discover MRs and merge them with target branch
+                }
+                gitLabTagDiscovery() // discover tags
+            }
+        }
+    }
+    // "Traits" ("Behaviours" in the GUI) that are NOT "declarative-compatible"
+    // For some 'traits, we need to configure this stuff by hand until JobDSL handles it
+    // https://issues.jenkins.io/browse/JENKINS-45504
+    configure { 
+        def traits = it / navigators / 'io.jenkins.plugins.gitlabbranchsource.GitLabSCMNavigator' / traits
+        traits << 'io.jenkins.plugins.gitlabbranchsource.ForkMergeRequestDiscoveryTrait' {
+            strategyId('2')
+            trust(class: 'io.jenkins.plugins.gitlabbranchsource.ForkMergeRequestDiscoveryTrait$TrustPermission')
+        }
+    }
+    // "Project Recognizers"
+    projectFactories {
+        workflowMultiBranchProjectFactory {
+            scriptPath 'Jenkinsfile'
+        }
+    }
+    // "Orphaned Item Strategy"
+    orphanedItemStrategy {
+        discardOldItems {
+            daysToKeep(10)
+            numToKeep(5)
+        }
+    }
+    // "Scan Organization Folder Triggers" : 1 day
+    // We need to configure this stuff by hand because JobDSL only allow 'periodic(int min)' for now
+    configure { 
+        it / triggers / 'com.cloudbees.hudson.plugins.folder.computed.PeriodicFolderTrigger' {
+            spec('H H * * *')
+            interval(86400000)
```

Comment:
Not sure this is necessary?
```
triggers {
  periodicFolderTrigger {
    interval('1d')
  }
}
```

Reply:
It is there in the JCasC demo. If what you say works we can remove this.

Reply:
it works what I tested 👍

Reply:
oh... It works for child items...